### DISCUSSION TITLE
Swagger documenation is not being generated for models that use gorm.model declarative

### DISCUSSION THREAD

**Discussion by ggoosen** [swaggo/swag#1864] [General]
If you use model like the one below, swag will not document it. 

type User struct {
	gorm.Model
	// Username of the user
	// @example johndoe
	Username string `json:"username"`
	// First name of the user
	// @example John
	FirstName string `json:"first_name"`
	// Last name of the user
	// @example Doe
	LastName string `json:"last_name"`
	// Email address of the user
	// @example john.doe@example.com
	Email string `json:"email"`
	// Password of the user (hashed)
	// @example (not shown for security reasons)
	Password string `json:"password" swaggerignore:"true"` // "-" excludes from JSON, swaggerignore excludes from Swagger docs
	// Role of the user
	// @example admin, user
	Role string `json:"role"`
}

2024/08/08 13:53:07 Generating models.User
2024/08/08 13:53:07 Error parsing type definition 'models.User': : cannot find type definition: gorm.Model
2024/08/08 13:53:07 Skipping 'models.User', recursion detected.
2024/08/08 13:53:07 Skipping 'models.User', recursion detected.

**Comment by et-do** [comment]
also having this issue
